### DISCUSSION TITLE
How to make a password protected page?

### DISCUSSION THREAD

**Discussion by nzyeehaa** [datenstrom/community#482] [Ask a question] [answered]
yellow has come a long way, the user access rights is great...

currently, 
if i have a user that i want to log in, and view a hidden menu and additional pages, 
I can do that by removing _Access: `create, edit, delete, upload`_ in _user.ini_

The problems I have..
- they can still access the _edit page_ screen (but nothing works)
- they can still access the _help_ file (which would only be for _create, edit, delete, upload_ people)
- the pages could theoretically still be searched by robots as there is nothing actually hiding them

is it worth having page `Status: draft / ignore / hidden / member`
and adjusting edit js to exclude `edit page` when they dont have _create/edit/delete_ rights
and adjusting edit.js to exclude `help` file when they dont have _create/edit/delete_ rights
... then finally - the menu may have to be altered to skip `member` pages(like it does for drafts)

any feedback?

**Comment by markseuffert** [comment, marked as answer]
There's an extension for that: [Private extension by Steffen Schultz](https://github.com/schulle4u/yellow-extensions-schulle4u/tree/main/private).
